### ISSUE TITLE
Building v0.6.11 failed test 1564

### ISSUE THREAD

**Issue by wdkrnls** [paulfitz/coopy#7]
I tried building the last tagged version and found that a test was failing:

failed test 1564 - column_rename_base_csv_reverseCompare (Failed)
Mismatch on cell 0,6: 'Throgs Nook' vs 'Throgs Neck'

Should I have expected this? I tried to install from an isolated environment (on GNU Guix) where only fossil, openssl, and zlib were available. This was the only test that failed.

**Comment by paulfitz** [comment]
@wdkrnls as you can tell from the commit history, I haven't poked at coopy for a long time - but I'd guess such a test failure is genuine. As to why, I don't have any special insight at this point, it has been too long - sorry!

**Comment by wdkrnls** [comment]
Fair enough. I suppose I will try and build daff instead :) I was under the mistaken belief it did not support sqlite. It's nice to learn I was wrong. Thanks!
